toolkit/resources/manifests/package/toolchain_x86_64.txt

```diff
@@ -87,16 +87,16 @@ flex-debuginfo-2.6.4-7.cm2.x86_64.rpm
 flex-devel-2.6.4-7.cm2.x86_64.rpm
 gawk-5.1.0-2.cm2.x86_64.rpm
 gawk-debuginfo-5.1.0-2.cm2.x86_64.rpm
-gcc-11.2.0-2.cm2.x86_64.rpm
-gcc-c++-11.2.0-2.cm2.x86_64.rpm
-gcc-debuginfo-11.2.0-2.cm2.x86_64.rpm
+gcc-11.2.0-3.cm2.x86_64.rpm
+gcc-c++-11.2.0-3.cm2.x86_64.rpm
+gcc-debuginfo-11.2.0-3.cm2.x86_64.rpm
 gdbm-1.21-1.cm2.x86_64.rpm
```

Comment:
Test build shows the following ("glibc-static") needs to be added to the manifests:
ERROR: Mismatched packages between '/mnt/vss/_work/1/s/toolkit/resources/manifests/package/toolchain_x86_64.txt' and '/mnt/vss/_work/1/s/build/toolchain/toolchain_built_rpms_all.tar.gz': 
glibc-static-2.35-3.cm2.x86_64.rpm

Reply:
I think this was for #3748?